src/fixes.rs

```diff
@@ -106,18 +106,18 @@ pub fn get_fixes_without_resolving_overlapping(
                     .iter()
                     .map(|import_path| format!("use {};\n", import_path))
                     .join("");
-                fixes
-                    .entry(location.file_id)
-                    .or_insert_with(Vec::new)
-                    .push(Fix {
-                        span: TextSpan {
-                            start: TextOffset::START,
-                            end: TextOffset::START,
-                        },
-                        suggestion: imports_suggestion,
-                        description,
-                    });
+                fix.suggestions.push(Suggestion {
```

Comment:
[nit]: I would hide the suggestions vector away from the user so he can't modify it in a potentially weird ways

Reply:
I think that if anybody wants to get them as mutable, it's theirs responsibility to manage it properly. I think making it private would result in methods like this later on https://github.com/starkware-libs/cairo/blob/4bb2f07f16e1eab975290f338eedb67e311ff643/crates/cairo-lang-filesystem/src/span.rs#L29